priv/templates/common/error.greenbar

```diff
@@ -1,23 +1,38 @@
-~attachment title="Command Error" color="#ff3333" Caller=$initiator Pipeline=$pipeline_text "Pipeline ID"=$id Started=$started~
-~if cond=$planning_failure ~
-The pipeline failed planning the invocation:
-~br~
+~if cond=$execution_failure != ""~
```

Comment:
Ooh, didn't know that! I'll go toss some comments in to make this template more clear and then merge it.